pkg/collector/corechecks/cluster/orchestrator/transformers/k8s/poddisruptionbudget_test.go

```diff
@@ -22,6 +23,13 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+func sortPodDisruptionBudgetFields(pdb *model.PodDisruptionBudget) {
+	if pdb.Metadata != nil {
+		sort.Strings(pdb.Metadata.Labels)
+	}
+	sort.Strings(pdb.Tags)
+}
+
```

Comment:
I think you can use `assert.ElementsMatch(t, tc.expect, got)`, this will match the elements no matter the order

Reply:
But then I have to compare fields manually, I wanted to avoid that:

```
--- FAIL: TestExtractPodDisruptionBudget (0.00s)
    --- FAIL: TestExtractPodDisruptionBudget/empty (0.00s)
        assertions.go:1107: 
            	Error Trace:	/Users/joachim.bartosik/go/src/github.com/DataDog/datadog-agent/pkg/collector/corechecks/cluster/orchestrator/transformers/k8s/poddisruptionbudget_test.go:301
            	Error:      	"metadata:<> spec:<> status:<> " has an unsupported type ptr, expecting array or slice
            	Test:       	TestExtractPodDisruptionBudget/empty
```